README.md

```diff
@@ -97,22 +97,29 @@ presents the detailed design of the controller.
 
 ## Roadmap
 
-This is a high-level overview of the main priorities for 2023, in expected order of release:
-
-- Cooperative preemption support for workloads that implement checkpointing [#477](https://github.com/kubernetes-sigs/kueue/issues/477)
-- Flavor assignment strategies, e.g. _minimizing cost_ vs _minimizing borrowing_ [#312](https://github.com/kubernetes-sigs/kueue/issues/312)
-- Integration with cluster-autoscaler for guaranteed resource provisioning
-- Integration with common custom workloads [#74](https://github.com/kubernetes-sigs/kueue/issues/74):
-  - Kubeflow (TFJob, MPIJob, etc.)
-  - Spark
-  - Ray
-  - Workflows (Tekton, Argo, etc.)
-
-These are features that we aim to have in the long-term, in no particular order:
-
+High-level overview of the main priorities for 2025:
```

Comment:
What do you think about adding DRA?

Reply:
Sure we can add, but maybe to the long-term goals, to be less commital?

Reply:
for now I added it to the bottom section